features/schedule/basic_workflow/feature.go

```diff
@@ -0,0 +1,122 @@
+package basic_workflow
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"go.temporal.io/api/enums/v1"
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk-features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+)
+
+var Feature = harness.Feature{
+	Workflows: BasicScheduleWorkflow,
+	Execute:   Execute,
+}
+
+func BasicScheduleWorkflow(ctx workflow.Context, arg string) (string, error) {
+	return arg, nil
+}
+
+func Execute(ctx context.Context, r *harness.Runner) (client.WorkflowRun, error) {
+	// Start schedule every 2s
+	workflowID := uuid.NewString()
+	handle, err := r.Client.ScheduleClient().Create(ctx, client.ScheduleOptions{
```

Comment:
Starting a workflow every 2 seconds can be pretty aggressive in a "non production level" test environment. Just in case something goes wrong and the schedule stays active longer than expected, I would set `limitedActions = true`, and `remainingActions = 2`, or `endAt = now + 2 minutes`...

Reply:
The schedule is only supposed to be valid for the life of this test in an isolated way. I delete the schedule before this returns.